src/platforms/hosted/serial_win.c

```diff
@@ -147,6 +147,27 @@ static char *find_bmp_by_serial(const char *serial)
 	return port_name;
 }
 
+static void print_aux_by_serial(const char *serial)
+{
+	char *serial_path = format_string("\\%s", serial);
+	if (!serial_path)
+		return;
+	char *prefix = read_key_from_path(serial_path, "ParentIdPrefix");
+	free(serial_path);
+	if (!prefix)
+		return;
+	char *parameter_path = format_string("&MI_02\\%s&0002\\Device Parameters", prefix);
+	if (!parameter_path) {
+		free(prefix);
+		return;
+	}
+	char *port_name = read_key_from_path(parameter_path, "PortName");
+	free(prefix);
+	if (!port_name)
+		return;
+	DEBUG_WARN("Found AUX Serial at %s\n", port_name);
+}
```

Comment:
This is missing just one thing - `free(port_name);` - so the function doesn't leak any memory - other than that this looks great!